monkey/monkey_island/cc/next_ui/src/_components/app-nav/app-drawer/drawer-header/DrawerHeader.tsx

```diff
@@ -5,29 +5,41 @@ import IconButton from '@mui/material/IconButton';
 import MonkeyHeadSvg from '@/assets/svg-components/MonkeyHead.svg';
 import SvgIcon from '@mui/material/SvgIcon';
 import React from 'react';
+import { styled } from '@mui/material/styles';
 
 export interface DrawerHeaderProps {
     onClose?: () => void;
 }
 
+const ThemedCloseIcon = styled(CloseIcon)(({ theme }) => ({
+    color: theme.palette.primary.contrastText
+}));
+
+const DrawerHeaderStyled = styled('div')(({ theme }) => ({
```

Comment:
These styles can be moved to a separate file. When it comes to styling in general we should follow the MUI styling guide, rather than whatever we're doing with `something.module.scss`. I'll create an issue

Reply:
I agree that styles should be easier to locate, and not scattered throughout the code. Having these styles in a file dedicated to styles is a step in the right direction